Makefile

```diff
@@ -85,7 +85,7 @@ include $(TPARTYDIR)/mshadow/make/mshadow.mk
 include $(DMLC_CORE)/make/dmlc.mk
 
 # all tge possible warning tread
-WARNFLAGS= -Wall -Wsign-compare
+WARNFLAGS= -Wall -Wsign-compare  -Wno-comment
```

Comment:
TBH I wish we were fixing this in ngraph instead of here, but this is okay enough IMO.